lib/plds_web/router.ex

```diff
@@ -4,6 +4,11 @@ defmodule PLDSWeb.Router do
 
   scope "/" do
     pipe_through [:fetch_session, :protect_from_forgery]
-    live_dashboard "/", metrics: PLDSWeb.Telemetry
+
+    get "/health", PLDSWeb.HealthController, :index
+
+    live_dashboard "/",
+      metrics: PLDSWeb.Telemetry,
```

Comment:
We can nuke the telemetry stuff I think. Can we set it to false or something to disable the page altogether?